process/block/metablockProposal.go

```diff
@@ -268,9 +266,7 @@ func (mp *metaProcessor) VerifyBlockProposal(
 	}
 
 	// TODO: move gas limit verification in baseProcessor
```

Comment:
can you remove also the todo as the initial proble (verify gas limit not being available for meta) is already done in this PR?